### ISSUE TITLE
Feature request - Some sort of web based dashboard

### ISSUE THREAD

**Issue by jrsphoto** [w0anm/Ekos-Indi_RPi_Image#2]
Love this script Chris, its really simple and does most everything for you.  Thank you for this.  So I've been going back and forth between rolling my own solution and Stellarmate, (and now your script).  One thing I love about Stellarmate is its web dashboard.  Its pretty functional and lets you manage a lot of things like updates, services, even some diagnostics via the log files.  I would be nice to have something like this.  You could even add buttons to enable/disable services like X11vnc or Window manager greeter.  - Just a thought.

One more question.. Are you scripts tied to ARM architecture?  I've got a thin-client computer with an i3 processor and I'm thinking of running this on it, if the script would work.  Should be possible to detect system architecture and install based on what it detects?

Thanks again Chris, this is really good.

-John

**Comment by jrsphoto** [comment]
I've been working on a dashboard..  take a look a this project, Its pretty simple.  https://github.com/femto-code/Raspberry-Pi-Dashboard

**Comment by w0anm** [comment]
> One more question.. Are you scripts tied to ARM architecture? I've got a thin-client computer with an i3 processor and I'm thinking of running this on it, if the script would work. Should be possible to detect system architecture and install based on what it detects?
 
There are some Arm unique area, such as boot and boot/firmware files.  This is not how x86 systems handle OS booting.  The packaging and some file add-ons could be unique to Raspberry pi.   You would need to go though the script and determine what is needed or use this as a guide.

**Comment by jrsphoto** [comment]
Thanks Chris.  I've got an i5 thin-client computer on the way and when it gets here I'll dive into this more.  I think I've got it figured out on how to make the script auto detect the system architecture using `dkpg --print-architecture`.  In the meantime, I'll fork the repo and have a go at it with WSL.

Btw, I also notice your in Minnesota!  I'm in Burnsville and if your in the area let me know!  I'm also N0CTL ;-)
